### ISSUE TITLE
Svelte Visualizer is always visible

### ISSUE THREAD

**Issue by bramus** [keycastr/keycastr#167]
When not casting the Svelte visualizer is visible. It would be handy if one could hide it when not casting.

**Comment by akitchen** [comment]
Yep, that's the idea. An option could be added but it's a pretty low priority TBH.
